src/Components/Address/AddressForm.tsx

```diff
@@ -112,6 +121,10 @@ export const AddressForm: React.FC<AddressFormProps> = ({
     }
     setKey(key)
     onChangeValue(key, ev.currentTarget.value)
+
+    if (userSelectedAddressOption && key !== "name" && key !== "phoneNumber") {
+      trackAutoCompleteEdit(key)
```

Comment:
Sorry, one more nit here - can we only track this once per selected address? 
<img width="356" alt="image" src="https://github.com/artsy/force/assets/9088720/17d92061-1190-4603-8a48-87539b442736">

Reply:
ah, maybe that makes sense - adding secondary info would mean something different than fixing the zip code. Your call - either track it the first time or once for each field, and we can confirm data's preference later (cc @daytavares)